pages/index.tsx

```diff
@@ -68,6 +72,7 @@ const HeroPage: React.FC = () => {
   }, []);
 
   useEffect(() => {
```

Comment:
@extremobemo do we need this here too? I see we've got very similar useEffect in the new Curtain component